docs-src/content/functions/coll.yml

```diff
@@ -136,6 +136,37 @@ funcs:
       - |
         $ gomplate -i '{{ .books | jsonpath `$..works[?( @.edition_count > 400 )].title` }}' -c books=https://openlibrary.org/subjects/fantasy.json
         [Alice's Adventures in Wonderland Gulliver's Travels]
+  - name: coll.JQ
+    alias: jsonpath
+    description: |
+      Filters an input object or list using the JQ language implemented by gojq (https://github.com/itchyny/gojq).
+
+      Any object or list may be used as input. The output depends somewhat on the expression; if multiple items are matched, an array is returned.
```

Comment:
Could a string be used as input too? If not that should be pointed out, and a workaround suggested (i.e. parse with `data.JSON` first, etc)

Reply:
Yes, any JSON datatype can be used as input, e.g. it works also for string or bolean as well:
```bash
$ gomplate -i '{{ "test" | jq `.` }}'
test
$ gomplate -i '{{ true | jq `.` }}'
true
```
I'll update the description as well.

Reply:
Ah - of course. It's probably also worth documenting the fact that string inputs are not first parsed as JSON then!